azure-cosmos/src/main/java/io/micronaut/azure/cosmos/client/CosmosClientConfiguration.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2017-2020 original authors
```

Comment:
Minor thing but worth knowing: instead of copying the license header from another file, just run `./gradlew spotlessApply` and it will automatically add it, with the correct year.